src/commands/ai/text-gen.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+
+class AiTextGenCommand extends BoxCommand {
+	async run() {
+		const { flags, args } = this.parse(AiTextGenCommand);
+		let options = {};
+
+        if (flags.dialogue_history) {
+            options.dialogueHistory = flags.dialogue_history;
+        }
+        options.prompt = flags.prompt;
+        options.items = flags.items;
+        let answer = await this.client.ai.textGen({
+            prompt: options.prompt,
+            items: options.items,
+            dialogue_history: options.dialogueHistory,
+        });
+
+        await this.output(answer);
+	}
+}
+
+AiTextGenCommand.description = 'Sends an AI request to supported LLMs and returns an answer specifically focused on the creation of new text.';
+AiTextGenCommand.examples = ['box ai:text-gen --dialogue_history=prompt="What is the status of this document?",answer="It is in review" --items=id=12345,type=file --prompt="What is the status of this document?"'];
+AiTextGenCommand._endpoint = 'post_ai_text_gen';
+
+AiTextGenCommand.flags = {
+	...BoxCommand.flags,
+    dialogue_history: flags.string({
+        required: false,
+        description: 'The history of prompts and answers previously passed to the LLM.',
+        multiple: true,
+        parse(input) {
+            const record = {};
+            for (const part of input.split(',')) {
```

Comment:
In dialogue-history, both the `prompt` and `answer` can contain the `,` character. In such cases this command ry will fail because of the split on this character. 
Have you considered another solution that would avoid this problem?